docs/pages/concepts/offloading/mutating-webhook.md

```diff
@@ -0,0 +1,91 @@
+
+```yaml
+clusterSelector:
+  nodeSelectorTerms:
+  - matchExpressions:
+    - key: liqo.io/region
+      operator: In
+      values:
+      - us-west-1
+```
+
+#### LocalAndRemote PodOffloadingStrategy
+
+The `LocalAndRemote` strategy allows Pods to be scheduled both on local nodes and a subnet of the virtual nodes.
+Hence, the webhook adds the *toleration* to the virtual nodes *taint*, as well as mutates the pod affinity introducing two (ORed) `NodeSelectorTerms`:
+
+* The first `NodeSelectorTerm` enforces the constraints specified through the `ClusterSelector`, and applies to virtual nodes.
+* The second `NodeSelectorTerm` enables Pod scheduling on all physical nodes, as they might not match the `ClusterSelector`.
+
+Hence, the resulting `NodeAffinity` constraint applied to a Pod would resemble the following:
+
+```yaml
+nodeAffinity:
+  requiredDuringSchedulingIgnoredDuringExecution:
+    nodeSelectorTerms:
+      - matchExpressions:
+        - key: liqo.io/region
+          operator: In
+          values:
+          - us-west-1
+      - matchExpressions:
+        - key: liqo.io/type
+          operator: NotIn
+          values:
+          - virtual-node
+```
+
+In case a given Pod already contains a set of `NodeAffinity` constraints, they are merged appropriately by the Liqo webhook with the ones above, ensuring that only the Nodes respecting the intersection of the restrictions can be selected as valid scheduling targets.
+
+#### Local PodOffloadingStrategy
+
+The `Local` strategy does not allow Pods to be scheduled on virtual nodes, hence behaving from this point of view as if Liqo were not present (although other resources, such as services, might be reflected to enable remote consumption).
+Consequently, in this case the webhook is not activated, and Pods are neither appended the *toleration* nor any additional *NodeAffinity* constraints.
+
+#### Remote PodOffloadingStrategy
+
+The `Remote` strategy forces Pods to be scheduled on virtual nodes only.
+Hence, the webhook adds the *toleration* to the virtual nodes *taint*, as well as mutates the pod affinity introducing a single `NodeSelectorTerm` with two (ANDed) `MatchExpressions`:
+
+* The first `MatchExpression` term enforces the constraints specified through the `ClusterSelector`.
+* The second `MatchExpression` ensures Pods are scheduled on virtual nodes only (i.e., with the `liqo.io/type=virtual-node` label).
+
+The resulting `NodeAffinity` constraint applied to a Pod resembles the following (in case it already contained a set of constraints, they would be merged accordingly):
+
+```yaml
+nodeAffinity:
+  requiredDuringSchedulingIgnoredDuringExecution:
+    nodeSelectorTerms:
+      - matchExpressions:
+        - key: liqo.io/region
+          operator: In
+          values:
+          - us-west-1
+        - key: liqo.io/type
+          operator: In
+          values:
+          - virtual-node
+```
+
```

Comment:
At the end of this section I would add a pointer to the proper liqoctl page that says how to set the above information through command line, just to help the reader to understand how this can be done (withouth having to manually modify the resources).